index.bs

```diff
@@ -4617,3 +4787,18 @@ Possible mitigations:
     presence from the reporting origin. Compared to the previous mitigation, the
     proxy server could itself handle the [=event-level report/trigger priority=]
     functionality, at the cost of increased complexity in the proxy.
+
+## Attribution Scope ## {#attribution-scope}
+
+*This section is non-normative.*
+
+Abusive actors can exploit the attribution scope feature by registering "ghost navigation sources" instead of event sources. During source registration, they can:
+1. Redirect to themselves and register [=attribution source/attribution scope limit=] - 1 navigations. 
+1. Initiate multiple background requests.
+1. Initiate a foreground request and a background request.
+
+This allows them to choose more complex configurations with different [=attribution source/attribution scopes=] (more bits) for their ghost registrations.
+
+Proposed mitigation: 
+1. Limit to 1 navigation source with non-empty [=attribution source/attribution scopes=] per navigation per reporting origin.
+1. Limit the number of sources a reporting origin can register per initiated navigation regardless of the attribution scopes feature, see <a href="https://github.com/WICG/attribution-reporting-api/issues/1128">issue 1128</a> for context.
```

Comment:
Does this match what's in #1282's `Additional Privacy Limits` section?

Reply:
Looking more closely, it seems that the explainer and spec differ here. The explainer talks about unique sets of scopes, while the spec talks about the number of sources.

I think we should also be a bit more clear on what the mitigation actually entails, even though we aren't fully spec'ing this yet due to the lack of integration with navigation in general. Specifically: Are extraneous sources dropped entirely, or are they registered without scopes, or something else?